index.html

```diff
@@ -56,5 +56,191 @@ <h1 class="Intro">
+        </div>
+
+        <div class="second--container">
+          <h1>Multi-Post Stories</h1>
+          <ul class="ul--list">
+            <li>CANOPY</li>
+            <li>Back End Dev</li>
+            <li>2015</li>
+          </ul>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required.
+          </p>
+          <ul class="skill--list">
+            <li>HTML</li>
+            <li>CSS</li>
+            <li>Javascript</li>
+          </ul>
+          <button type="button" class="see--project">
+            <a class="see--project--link" href="#">See project</a>
+          </button>
+        </div>
+      </div>
+    </section>
+
+    <section class="about--section">
+      <div>
+        <h2>About Myself</h2>
+        <p class="about--text">
+          Hello I'm a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don't hesitate to
+          contact me.
+        </p>
+        <p class="hero-connect-text">LET'S CONNECT</p>
+        <ul class="hero-icon-container">
+          <li>
+            <a href="https://twitter.com/AyokunnumiA">
+              <img src="images/Vector.svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/ayokunnumiomololu">
+              <img src="images/Linkedin icon.svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://www.linkedin.com/in/ayokunnumiomololu">
+              <img src="images/Vector (1).svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/AyokunnumiA">
+              <img src="images/Vector (2).svg" alt="" />
+            </a>
+          </li>
+          <li>
+            <a href="https://twitter.com/AyokunnumiA">
+              <img src="images/Vector (3).svg" alt="" />
+            </a>
+          </li>
+        </ul>
+        <button type="button" class="see--project">
+          <a class="see--project--link" href="#">Get my resume</a>
+        </button>
+      </div>
+      <div>
+        <div class="dropdown">
+          <p>
+            <span>Languages</span>
+            <img src="images/arrow.png" alt="" />
+          </p>
+          <ul class="dropdown-list">
+            <li>
+              <img src="images/js.svg" alt="" />
+              <span>Javascript</span>
+            </li>
+            <li>
+              <img src="images/html.svg" alt="" />
+              <span>HTML</span>
+            </li>
+            <li>
+              <img src="images/css.svg" alt="" />
+              <span>CSS</span>
+            </li>
+          </ul>
+        </div>
+        <div class="dropdown">
+          <p>
+            <span>Frameworks</span>
+            <img src="images/arrow.png" alt="" />
+          </p>
```

Comment:
- [ ] To align with the suggested [Figma design,](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?type=design&node-id=39-122&t=zg3xfHFYtcudEhRh-0) it is recommended to use the right arrow `>` sign instead of a downward arrow.
- [ ]  This change ensures consistency with the design and enhances the overall visual appeal of your project. 
- [ ] By using the right arrow `>` sign, you maintain alignment with the suggested design and create a more cohesive user experience.🤔🤔